.eslintignore

```diff
@@ -0,0 +1 @@
+coverage/*
```

Comment:
Rather that having another admin file, I was opting for "ignorePatterns" in the eslintrc.json file. I'm going to merge this in, and then revert to using that if you don't mind?